recipes/newrelic/infrastructure/alerts/golden.yml

```diff
@@ -86,6 +85,14 @@ install:
               -L -H 'Content-Type: application/json' \
               -d @/tmp/policy.json
           )
+
+          # Check if got error while try to create policy
+          POLICY_RESULT_ERR=$(echo $POLICY_RESULT | /usr/local/bin/newrelic utils jq '.|select(.errors != null)|.errors[0].message ')
+          if [[ ! -z "$POLICY_RESULT_ERR" ]]; then
+            >&2 echo 'Could not create a new alert policy for {{.ALERT_POLICY_NAME}} got the following error:' "$POLICY_RESULT_ERR"
```

Comment:
@NRhzhao I think if we redirect to stderr we're actually not getting to see the error message. Given this makes little difference, I think it would be better to display in stdout, instead of trying to send to the error stream.
There are other similar redirect below.